src/en/models-cmr-scene-2.md

```diff
@@ -244,104 +257,32 @@ ian:admin/default.hosted-mysql:
       access: read
 ```
 
-To find offers in a specified model:
+To find offers in model `cmr-model-1` on controller `lxd-cmr-1`:
 
 ```bash
-juju find-offers admin/default
-juju find-offers ian:admin/default
+juju find-offers lxd-cmr-1:admin/cmr-model-1
 ```
 
-To find offers with a specified interface on the current controller:
+## Relating to offers from behind a firewall
 
-```bash
-juju find-offers --interface mysql
-```
+Let the consuming model in this scenario be protected by a firewall that NATs
+all outgoing traffic to the single IPv4 address of 69.32.56.10/32.
 
-To find offers with a specified interface on a specific controller:
+Now request to the offering side to allow this address to contact the offer:
 
 ```bash
-juju find-offers --interface mysql ian:
+juju add-relation mediawiki:db lxd-cmr-1:admin/cmr-model-1.mysql --via 69.32.56.10/32
 ```
 
-To find offers with "sql" in the name:
-
-```bash
-juju find-offers --offer sql ian:
-```
-
-### Relating to offers from behind a firewall
-
-Sometimes, the consuming application is deployed behind a firewall where NAT is
-used, such that traffic egresses via a different address/network to that on
-which the consuming application is hosted.
-
-In this case, the relate --via option is used to inform the offering side so
-that the correct firewall rules can be set up.
+This will work providing the offering side did not previously set up a
```

Comment:
I find this paragraph confusing. Does this mean that it will work if no whitelist has been set?

Reply:
I agree I can make it clearer. The answer to your question is yes, without a whitelist any request for a firewall rule will be accepted.

Reply:
This is done, and with less words.